Formula/mpd.rb

```diff
@@ -108,6 +108,7 @@ def install
     system "make", "install"
 
     (etc/"mpd").install "doc/mpdconf.example" => "mpd.conf"
+    ohai "When launching as a service with a non-standard config location, append '<string>$mpdconfigpath</string>' before the closing array tag in ~/Library/LaunchAgents/#{homebrew.mxcl.mpd}.plist"
```

Comment:
If this is Homebrew-specific information, it should be in the `caveats`.  Otherwise let's remove it.